.gitignore

```diff
@@ -16,4 +16,4 @@ config.json
 */*/._*
 coverage.*
 lib-cov
-
+*.lock
```

Comment:
What's that one ?

Reply:
Was ignoring lock files since nobody seems to commit them to repo's anyways

Reply:
Then it should be package-lock.json, there's no file with lock extension like that modification suggested.

Reply:
Yarn uses `yarn.lock` ✌️

Reply:
I don't use yarn and the hapi org doesn't support it officially, no reason for me to include it there.

Reply:
Fair enough, lets just ignore this case then. works for me, I ignored it locally instead.

Reply:
@henrikdahl check out this [issue](https://github.com/yarnpkg/yarn/issues/3614) for yarn specific guidance.

Even if projects aren't committing their package-lock.json/yarn.lock files, [npm](https://docs.npmjs.com/files/package-locks) and [yarn](https://yarnpkg.com/lang/en/docs/yarn-lock/#toc-check-into-source-control) heavily recommend that they are tracked in repositories.